docs/pages/eas-update/debug.mdx

```diff
@@ -1,69 +1,87 @@
+### Unexpected runtime version
 
-In each, we expect to see configuration for the EAS Update URL and the runtime version. Here are the properties we'd expect to see in each file:
+<ImageSpotlight
+  alt="Deployment has no updates"
+  src="/static/images/eas-update/deployments-wrong-runtime.png"
+  style={{ maxWidth: 600 }}
+/>
 
-**AndroidManifest.xml**
+If the deployment runtime version is unexpected, it means your build was not built with the correct runtime version. To fix this, [configure your runtime version](/#configure-runtime-version).
 
-```xml
-...
-<meta-data android:name="expo.modules.updates.EXPO_RUNTIME_VERSION" android:value="your-runtime-version-here"/>
-<meta-data android:name="expo.modules.updates.EXPO_UPDATE_URL" android:value="https://u.expo.dev/your-project-id-here"/>
-...
-```
+### Unexpected branch
 
-**Expo.plist**
+<ImageSpotlight
+  alt="Deployment has no updates"
+  src="/static/images/eas-update/deployments-wrong-branch.png"
+  style={{ maxWidth: 600 }}
+/>
 
-```xml
-...
-<key>EXUpdatesRuntimeVersion</key>
-<string>your-runtime-version-here</string>
-<key>EXUpdatesURL</key>
-<string>https://u.expo.dev/your-project-id-here</string>
-...
-```
+If the deployment has an unexpected branch, you will need to [map your channel to the correct branch](/#map-channel-to-branch).
 
-## EAS Update configuration
+### Missing updates
 
-To debug the state of EAS Update in our project, we'll need to look at multiple spots in the system. Below is a diagram of how EAS Update works and the spots that are useful to inspect when finding the root cause of an issue. In the sections following, we'll inspect and verify these spots and more.
+<ImageSpotlight
+  alt="Deployment has no updates"
+  src="/static/images/eas-update/deployments-no-updates.png"
+  style={{ maxWidth: 600 }}
+/>
 
-<ImageSpotlight alt="Map of debugging spots" src="/static/images/eas-update/debug-map.png" />
+The displayed deployment does not have any updates. To fix this, [publish an update to the branch](/#publish-update).
 
-### 1. Verify a channel
+### Missing branch
 
-Builds have a property named `channel`, which EAS Update uses to link to a branch. A channel is often given to multiple platform-specific builds. For instance, we might have an Android build and an iOS build, both with a channel named `"production"`.
+<ImageSpotlight
+  alt="Deployment has no updates"
+  src="/static/images/eas-update/deployments-no-branch.png"
+  style={{ maxWidth: 600 }}
+/>
+
+The displayed deployment has the correct channel, but it is not linked to a branch. To fix this, [map the channel to the correct branch](/#map-channel-to-branch).
+
+### Missing deployment
 
-#### Configuring channels
+If your deployment is not displayed, this means your build is not configured properly with EAS Update. To fix this, [configure your channel](/#configure-channel) and [configure your runtime version](/#configure-runtime-version).
```

Comment:
```suggestion
If your deployment is not displayed, this means your build is not configured properly for EAS Update. To fix this, [configure your channel](/#configure-channel) and [configure your runtime version](/#configure-runtime-version).
```

Maybe mention as well to verify the general EAS Update configuration (updates.url)?